samples/posix/philosophers/prj.conf

```diff
@@ -0,0 +1,25 @@
+CONFIG_STDOUT_CONSOLE=n
+CONFIG_ASSERT=y
+CONFIG_ASSERT_LEVEL=2
+CONFIG_NUM_COOP_PRIORITIES=29
+CONFIG_NUM_PREEMPT_PRIORITIES=40
+CONFIG_SCHED_SCALABLE=y
+CONFIG_MP_MAX_NUM_CPUS=1
+
+CONFIG_LOG=y
+CONFIG_LOG_MODE_MINIMAL=y
+
+CONFIG_POSIX_API=y
+CONFIG_THREAD_STACK_INFO=y
+CONFIG_DYNAMIC_THREAD=y
+
+CONFIG_DYNAMIC_THREAD_POOL_SIZE=6
+CONFIG_MAX_PTHREAD_COUNT=6
+CONFIG_MAX_PTHREAD_MUTEX_COUNT=7
+
+#Enable thread awareness for debugging tools supporting it
+CONFIG_DEBUG_THREAD_INFO=y
+
+# CONFIG_SAMPLE_ERROR_CHECKING=y
+# CONFIG_PTHREAD_MUTEX_LOG_LEVEL_DBG=y
+# CONFIG_PTHREAD_LOG_LEVEL_DBG=y
```

Comment:
Leftover?

Reply:
Well, the original sample had a commented config option for debug purposes. I thought I would leave these here as well as hints in case someone needs them.

Actually, no debugging was necessary for this - it Just Worked (TM)